pkg/build/nodeimage/build_impl.go

```diff
@@ -133,23 +87,34 @@ func (c *buildContext) buildImage(dir string) error {
 	}
 
 	// copy artifacts in
-	if err = execInBuild("rsync", "-r", "/build/bits/", "/kind/"); err != nil {
```

Comment:
is this a win? moving from `rsync` to `docker cp`?

Reply:
yes and no. rsync on a volume mount might? be faster. but what's done here is less generic and simpler if you consider beyond this line.

but now we are not setting up this complex temp directory thing, it only remains for exporting tarballs from pulled images.

instead of syncing in a directory tree and setting up symlinks into path, we just move executables into path and make them executable, and images loaded directly. specific speciality files written specially (like version).

we can explore doing the equivilant of this build without docker commit later now.